### ISSUE TITLE
googlenet structure problem

### ISSUE THREAD

**Issue by kaleid0** [weiaicunzai/pytorch-cifar100#96]
in googlenet.py line45，"nn.BatchNorm2d(n5x5, n5x5)", it seems eps is incorrectly set to n5x5
